common/list/lists-mws-en.html

```diff
@@ -0,0 +1,241 @@
+---
+{
+	"title": "Lists for MWS",
```

Comment:
The title should not include MWS. It need to represent the functional advantage of this new notation which is not connected to a propriety product.

Reply:
Done. Thanks